src/main/groovy/grails/plugins/cacheheaders/CacheHeadersTrait.groovy

```diff
@@ -7,10 +7,8 @@ import groovy.transform.*
 @CompileStatic
 trait CacheHeadersTrait extends ServletAttributes {
 
-	@Autowired
```

Comment:
I guess to say "assumption" is to understate what I've seen.  Cache-Headers tests pass given the change, and functional tests that I've been running on another plugin (which depend on the Cache-Headers trait calls to service) also appear to be routing through the service properly.